script/build

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+set -e -o pipefail
```

Comment:
Forgive my ignorance but I was reading up on how `set` worked and came across [this blog post](https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/). I'm guessing since your use of `trap` below isn't for `ERR` it's not necessary but we don't want `-E` as well do we?